### ISSUE TITLE
Table component accessibility - sort button issue - (AVT 3)

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#287]
JAWS only reads the sort order when the user first focuses on the sort button for a column.
So once the user executes the sort, JAWS does not read the new sort order.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

Table component

> What did you expect to happen? What happened instead? What would you like to see changed?

JAWS should read out the sort order which will be applied when the user executes the sort, and once executed, JAWS should read the new sort order.

> What browser are you working in?

Firefox 63.0.3 (64-bit)

> What version of the Carbon Design System are you using?

"carbon-components": "9.53.2",
"carbon-components-angular": "2.0.8",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

**Comment by cal-smith** [comment]
So it seems like we're updating the `aria-label` and `alt` appropriately ... we may be able to do something with `aria-live` to work around it, but outside of that I don't see an issue with our markup.

**Comment by johnharan** [comment]
@cal-smith @zvonimirfras Hi guys, I'm still not hearing JAWS read the new sort order. JAWS reads the initial sort order when I tab to the sort button, however once I press enter to execute the sort the new sort order isn't announced, i just hear JAWS say "Enter". Maybe as you mentioned aria-live could work.

**Comment by cal-smith** [comment]
Weird, it was definitely working for me ... I'll double check today :+1:

**Comment by cal-smith** [comment]
So I couldn't get it to not work locally ... I'm opening a PR to add `aria-live` since it _should_ help, but I'll close this issue again for now.
